understanding/conformance.html

```diff
@@ -528,28 +436,22 @@ <h2>Understanding Conformance Claims</h2>
 &lt;/div&gt;
      </pre>
                   
-                  <p>Sometimes, one may want to make a claim for just the content that was added after
-                     a certain date. Or, one may want to claim WCAG 1.0 conformance for content up to a
-                     date and WCAG 2.0 for content that was created or modified after that date. There
-                     are no prohibitions in WCAG 2.0 to any of these practices as long as it is clear which
-                     pages are claiming conformance to which version of WCAG.
+                  <p>ある日付以降に追加されたコンテンツだけに適合宣言を行いたいことがあるかもしれない。あるいは、ある日付までのコンテンツには WCAG 1.0 への適合宣言を行い、その日付以降に制作又は更新したコンテンツには WCAG 2.0 への適合宣言をしたいこともあるかもしれない。WCAG 2.0 では、どのページが WCAG のどのバージョンに対して適合宣言をしているのかが明確である限り、そういった適合宣言を禁止していない。
```

Comment:
WCAG 2.0 → WCAG 2 とします?

```suggestion
                  <p>ある日付以降に追加されたコンテンツだけに適合宣言を行いたいことがあるかもしれない。あるいは、ある日付までのコンテンツには WCAG 1.0 への適合宣言を行い、その日付以降に制作又は更新したコンテンツには WCAG 2.0 への適合宣言をしたいこともあるかもしれない。WCAG 2 では、どのページが WCAG のどのバージョンに対して適合宣言をしているのかが明確である限り、そういった適合宣言を禁止していない。
```

Reply:
原文の最新版でそうなっていることもあり、気持ち悪さはありますが、ママとしてもらえれば…
https://www.w3.org/WAI/WCAG22/Understanding/conformance#conformance-claims

Reply:
了解です。原文がアレということで原文のママで。